htdocs/user/param_agendaReminder.php

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
Introducing a new tab to add few user options on interface for a user may lead to an overload of tabs. 
I recommend instead to put the setup parameters into the existing tab "User interface" (htdocs/user/param_ihm.php)